webpack.config.js

```diff
@@ -57,8 +59,17 @@ const config = {
       filename: 'multi_org.html'
     }),
     new ExtractTextPlugin({
-      filename: 'styles.[chunkhash].css'
-    })
+      filename: 'styles.[chunkhash].css',
+      publicPath: '/assets/'
+    }),
+    new ManifestPlugin({
+      fileName: 'asset-manifest.json'
+    }),
+    new CopyWebpackPlugin([
+      { from: 'client/app/assets/robots.txt' },
+      { from: 'client/app/assets/css/login.css', to: 'styles/login.css' },
+      { from: 'client/app/assets/js/jquery.min.js', to: 'js/jquery.min.js' },
```

Comment:
Do we really need to store `jquery.min.js` in `app/assets` folder? We already have jquery and all pre-built bundles in `node_modules`

Reply:
Probably not. But then we need to create a new entry point that will export only jQuery?

Reply:
Can you try to replace `from` path to be `node_modules/jquery/dist/jquery.min.js`? Will it work?

Reply:
Brilliant. Done 👍 96ad3aaa259b8f97844eb0b9722b02c45ed970b4